SWLOR.Game.Server/Enumeration/AttackType.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace SWLOR.Game.Server.Enumeration
+{
+// This array is used to signal the preferred attack type of an ability to the native code that manages attack roll overrides.
+// Set the local variable ATTACK_TYPE_OVERRIDE on the attacking creature to one of the valid values below.  This is a one-shot:
+// the value of the variable will be wiped when it is read. 
+//
+// If this override is used, the equipped weapon and any related feats etc. will be ignored.  To make a special weapon that uses
+// (say) Force to attack, build the weapon in the toolset and give it the ATTACK_TYPE_OVERRIDE int variable with the same 
```

Comment:
I think I'd prefer to do this via weapon type, item property, or a combination thereof. The local variables are very hard to see and maintain when you're building stuff. What do you think?

Reply:
Weapon type is by default - you only need the variables if you want to override the default.  Item properties could work for items, but not for abilities - and there's no other way I can see to correspond information from a feat script running in the main engine to the native code.  But if I've missed something shout!